inginious/frontend/tasks.py

```diff
@@ -102,6 +111,19 @@ def get_context(self, language):
         return ParsableText(vals[0], "rst", self._translations.get(language, gettext.NullTranslations())) if len(vals) else \
             ParsableText(context, "rst", self._translations.get(language, gettext.NullTranslations()))
 
+    def get_tutorial(self):
+        """ Get the task tutorial on RST"""
+        parseTutorial = ParsableText(self._tutorial,"rst");
+        return parseTutorial
+
+    def get_solution_content(self):
+        """ Get the solution of the task """
+        return self._solution
+
+    def get_solution_language(self):
+        """ Get the language of the solution """
+        return self._solution_language
+
```

Comment:
This is not necessary. Send these variables directly to the html template as a parameter when the `task_editorial_preview.html` file is called in the function `editorial_task_preview`.